docs/contributing.rst

```diff
@@ -69,6 +69,45 @@ found in the `virtualenv docs`_.
 
 .. _`virtualenv docs`: https://virtualenv.pypa.io
 
+Running certbot against a locally deployed Pebble CA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By following the instructions above, you can execute Certbot against a local instance
+of Pebble CA. This is useful to verify that the modifications done to the code makes
+Certbot behave as expected.
+
+You need Docker installed, and a user with access to the Docker client.
+
+The virtual environment set up with `python tools/venv.py` contains two commands
+that can be used once the virtual environment is activated:
+
+.. code-block:: shell
+
+    run_acme_server
+
+- Start a local instance of Pebble CA, and keep displaying its log on foreground.
+- Press CTRL+C to stop this instance.
+- This instance is configured to validate challenges against a certbot executed locally.
+
+.. code-block:: shell
+
+    certbot_tests [ARGS...]
+
+- Execute certbot with the provided ARGS in test mode: verbose output, debug enabled...
```

Comment:
I'd personally get rid of the phrase "test mode" and say something like:

> Execute Certbot with the provided arguments and other arguments useful for testing such as verbose output, full tracebacks in case Certbot crashes, etc.

Reply:
Improved the sentence with your advices.